Taskfile.yml

```diff
@@ -15,7 +15,4 @@ tasks:
     desc: Sync the latest tag with GoDocs
     cmds:
       - echo "syndicating to GoDocs..."
-      - test $(GIT_DOMAIN)
-      - test $(REPO)
-      - test $(VERSION_SHORT)
-      - curl https://proxy.golang.org/$(GIT_DOMAIN)/$(REPO)/@v/$(VERSION_SHORT).info
+      - curl "https://proxy.golang.org/github.com/$REPO/v/$VERSION_SHORT.info"
```

Comment:
I'm still thinking, that maybe we could have this command directly in gh action instead of here?
As I don't see any reason when would we like to run it without gh action